src/parsers/manifestjson.js

```diff
@@ -1088,6 +1088,49 @@ export default class ManifestJSONParser extends JSONParser {
     }
   }
 
+  getExperimentApiPaths() {
+    const { experiment_apis } = this.parsedJSON;
+
+    if (!experiment_apis) {
+      return [];
+    }
+
+    // We need to build a list of API "paths" for each registered experimental
+    // API. The data in the `manifest.json` would look like this:
+    //
+    // "experiment_apis": {
+    //   "some-name": {
+    //     "schema": "experiments/some-name/schema.json",
+    //     "parent": {
+    //       "scopes": ["addon_parent"],
+    //       "script": "experiments/some-name/api.js",
+    //       "paths": [["some", "name"]]
+    //     }
+    //   }
+    //
+    // We are interested in the `paths` array (of array), which contains API
+    // "paths". We need to get each entry for each experiment and we build API
+    // "paths" like:
+    //
+    // ['some.name']
+    //
+    // We could have either a "parent" or "child" property for each API, or
+    // both although it is less common.
+    //
+    const apiPaths = new Set();
+    for (const key of Object.keys(experiment_apis)) {
+      const { child, parent } = experiment_apis[key];
+      const parentPaths = parent?.paths ?? [];
+      const childPaths = child?.paths ?? [];
+
+      [...parentPaths, ...childPaths]
+        .filter((p) => Array.isArray(p) && p.length)
+        .forEach((p) => apiPaths.add(p.join('.')));
+    }
+
+    return Array.from(apiPaths.values());
```

Comment:
Nit, `return Array.from(apiPaths);` would also get us the same result, or maybe we may even just leave it as a Set?